cypress/integration/sign_in.spec.js

```diff
@@ -15,10 +15,12 @@ describe('Sign in flow', () => {
     });
 
     it('Has no detectable a11y violations on sign in page with error', () => {
-      cy.visit('/signin');
+      cy.visit(
+        '/signin?error=error&error_description=oh no somethings gone wrong',
+      );
```

Comment:
This related?

Reply:
ah it's not! crept in with the cherry-pick, let me change that

```suggestion
      cy.visit(
        '/signin',
      );
```